src/note.h

```diff
@@ -44,35 +43,32 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  * */
 
 enum ecolor {
-   COLOR_ERROR    = -1, /* error parser                        */
-   COLOR_OPTIONAL = 0,  /* the user did not indicate the color */
-   COLOR_OK       = 1,  /* the user indicate the color         */
+    COLOR_ERROR = -1, /* error parser                        */
+    COLOR_OPTIONAL = 0, /* the user did not indicate the color */
+    COLOR_OK = 1, /* the user indicate the color         */
 };
 
-struct __scrotnote
-{
-   char *font;    /* font name                   */
-   char *text;    /* text of the note            */
-   int x;         /* position screen (optional)  */
-   int y;         /* position screen (optional)  */
-   double angle;  /* angle text (optional)       */
-
-   struct color   /*                 (optional)  */
-   {
-      enum ecolor status;
-      int r,                  /* red             */
-          g,                  /* green           */
-          b,                  /* blue            */
-          a;                  /* alpha           */
-   } color;
+struct __ScrotNote {
+    char* font; /* font name                   */
+    char* text; /* text of the note            */
+    int x; /* position screen (optional)  */
+    int y; /* position screen (optional)  */
+    double angle; /* angle text (optional)       */
+
+    struct color /*                 (optional)  */
```

Comment:
Maybe the brace character `{` should start on the line of the name of struct

Reply:
sorry I'm not following you. The brace character `{` is on the same line of the name of the struct. Can you suggest what you want to see?

```
struct __ScrotNote {
```

Reply:
I meant in
`struct color {}`
If you agree I can modify it via the web editor.
Or if you make any changes, please do not force the push-commit because we cannot follow the changes and we will have to review everything again.

Reply:
I see. Sure, you can go ahead and correct it via the web editor. 

Sorry about the forced push.